src/components/Calculator.js

```diff
@@ -6,45 +11,58 @@ class Calculator extends Component {
   // eslint-disable-next-line no-useless-constructor
   constructor(props) {
     super(props);
-    this.state = {};
   }
 
-    createDigits = () => {
-      const digits = [];
-      let iterate = 9;
-      while (iterate > 0) {
-        digits.push(<button>{iterate}</button>);
-        iterate -= 1;
-      }
-      return digits;
+  createDigits = () => {
+    // eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Same here, Please remove all the `eslint `disablement lines and fix eslint error in your code.

Reply:
Fixed.